src/wallet/wallet.h

```diff
@@ -54,6 +54,10 @@ void StopWallets();
 //! Close all wallets.
 void UnloadWallets();
 
+//! Unloads and deletes the wallet.
+//  Blocks current thread until the wallet is deleted.
+void UnloadWallet(std::shared_ptr<CWallet>&& wallet);
```

Comment:
Comment could say that it's not necessary to call this function unless you need to block waiting the unload.

Reply:
Will improve comment.